src/bigint.hpp

```diff
@@ -38,66 +38,50 @@ namespace libbig {
       std::string number;
 
     public:
-      largeInt() {
-      }
-
-      largeInt(int) {
-      }
-      largeInt(int64_t) {
-      }
-      largeInt(long long) {
-      }
-      largeInt(std::string) {
-      }
+      largeInt();
+
+      largeInt(int);
```

Comment:
Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285290)